docs/clustering/clustering_evaluation.md

```diff
@@ -48,7 +48,7 @@ kmeans = KMeans(random_state=10, n_init=5)
 
 ##### Elbow curve
 
-Elbow curve helps to identify the point at which the plot starts to become parallel to the x-axis. The K value corresponding to this point is the optimal number of clusters. In the below plot one is likely to select k=4.
+Elbow curve helps to identify the point at which the plot starts to become parallel to the x-axis. The K value corresponding to this point is the optimal number of clusters. In the below plot one is likely to select k=4. Ensure that the second parameter is Kmeans, MiniBatchKmeans, or BisectingKMeans to avoid errors.
```

Comment:
We can phrase this one better.

> `Ensure that the second parameter is Kmeans, MiniBatchKmeans, or BisectingKMeans to avoid errors.` ->

`Currently the `kmeans` argument input only accepts `Kmeans`, `MiniBatchKMeans`, and `BisectingKMeans`.